src/gp_instance.cpp

```diff
@@ -174,3 +175,63 @@ void GPInstance::EstimateSBNParameters() {
   double marginal_log_lik = GetEngine()->GetLogMarginalLikelihood();
   std::cout << std::setprecision(9) << marginal_log_lik << std::endl;
 }
+
+RootedTreeCollection GPInstance::GetRootedTreeCollection() {
+
+  Tree::TreeVector tree_vector;
+  Node::NodePtrVec trees = dag_.GenerateAllTrees();
+  EigenVectorXd bl = engine_->GetBranchLengths();
+
+  // Leaves encoding to parent subsplit encoding.
+  std::unordered_map<const Node*, Bitset> leaves2subsplit_indexer;
```

Comment:
"to" rather than "2" to be consistent with the rest of the code, please.

Reply:
Done.